patches/react-native-aes-crypto+2.1.1.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/react-native-aes-crypto/android/src/main/java/com/tectiv3/aes/RCTAes.java b/node_modules/react-native-aes-crypto/android/src/main/java/com/tectiv3/aes/RCTAes.java
```

Comment:
Super interesting to see how easy it is to patch a library.
This raises some questions though! Do you have any documentation you could share about patching?

- Will it be applied only if we use ver `2.1.1` of the library?
- What if we update the underlying library?
- How can we test this?

Reply:
The tool that was used was this: https://www.npmjs.com/package/patch-package.

Tha patches found in the `/patches` directory are automatically applied when installing the deps (`npm` and `yarn` pick them up).

In the _"Benefits of patching over forking"_ section, the docs mention:

> - Get told in big red letters when the dependency changed and you need to check that your fix is still valid.

So, to answer your first questions, it seems that it will always be applied and if the deps change it will show in the console so that we can check if it still works and/or update the patch.

How can we test this? :/ That's a good question...

Reply:
Ok, that's good to know, thanks!

For now, as it's the only patch we have, I guess we can make sure we properly read the big red letters and manually test the impacted feature when updating the dep...